model/project_ref.go

```diff
@@ -1261,11 +1261,6 @@ func (projectRef *ProjectRef) Upsert() error {
 	return err
 }
 
-// ProjectRef returns a string representation of a ProjectRef
-func (projectRef *ProjectRef) String() string {
```

Comment:
what's the reason for deleting this?

Reply:
Oh right--I think it's confusing to have a String() method when we have both ID and Identifier, and sometimes one is useful and sometimes it's not. This isn't an interface for anything as far as I'm aware; we shouldn't need to support it. I think it's better to just explicitly use Id / Identifier so we don't have this indirection

Reply:
it's not an interface that we have, but the reason some random structs have String() is to implement go's stringer interface which tells various things (including grip) what to print. It might still be worth removing which will make grip print the entire struct, but worth thinking about

Reply:
That makes sense but I would still like to remove it to avoid confusion, and we don't rely on String() for most of our other large structs. Is there something I can grep for in the code base to find places where we're logging project and relying on this? In what format would you pass something to grip such that it knows to use String() and not print the whole struct?

Reply:
it'll be hard to find I think. If you just grip.Info a project ref or task, you should see that it prints the string rather than the entire struct